### ISSUE TITLE
Path parameter not being passed correctly

### ISSUE THREAD

**Issue by ghost** [bixlabs/loopback-testing-helpers#1]
Hi, and thank you for this awesome tool!
I've found a little issue when passing path parameters to a remote method when using that method several times on the same test file.

Take this for example

I've got an endpoint which is defined like this:

```
  Model.addOne = (number, cb) => {
    var plusOne = number + 1;
    cb(null, plusOne);
  };

  Model.remoteMethod('addOne', {
    description: 'Adds a digit to a number',
    accepts: [
      {
        arg: 'number',
        type: 'number',
        required: true,
        description: 'number to add a digit',
        http: { source: 'path' },
      }
    ],
    returns: { arg: 'data', root: true },
    http: {
      path: '/:number/addOne',
      verb: 'post'
    }
  });
```

And on my tests files I've got

```
const app = require('../../server/server.js');
const assert = require('chai').assert;
const h = require('loopback-testing-helpers')(app);

describe('Addition', () => {
  before((done) => {
    app.on('booted', done);
  });

  it('Sum of 1 + 1 equals 2', (done) => {
    h.Subscription
      .addOne({ number: 1 })
      .expect(200)
      .expect((res) => {
        var data = res.body;
        assert.equal(data, 2);
      })
      .end(done);
  });

  it('Sum of 2 + 1 equals 3', (done) => {
    h.Subscription
      .addOne({ number: 2 })
      .expect(200)
      .expect((res) => {
        var data = res.body;
        assert.equal(data, 3);
      })
      .end(done);
  });
});
```

Result:
![image](https://cloud.githubusercontent.com/assets/17278273/22601069/10f08b0c-ea1c-11e6-9318-2dcc20fccd0e.png)

I've found that this happens because when this line https://github.com/bixlabs/loopback-testing-helpers/blob/master/index.js#L24 executes for the first time for that method the path parameter will be modified already.

1. /Model/:number/addOne
2. /Model/1/addOne

Now the next time the method is executed the key `:number` won't exist in the string thus failing to pass the new parameter value

Solution:

- Create a new temporary url that will point to the url

```
      var newUrl = url;

      var replaceKeys = function(key) {
        var keyValue = args[key];
        newUrl = newUrl.replace(':' + key, keyValue);
      };
```

I will submit a PR :smile:

**Comment by rkmax** [comment]
@evasquez26 thank you so much for this fix I'll merge and publish a new release in shortly

**Comment by rkmax** [comment]
fixed by #4
